### ISSUE TITLE
Localization needed

### ISSUE THREAD

**Issue by wanderingstan** [OriginProtocol/origin#1873]
Keeping a list of known localization issues:

- [ ] "Loading..." on main page
![image](https://user-images.githubusercontent.com/673455/55497598-f0b5a100-55fe-11e9-8a24-eadce491a5ec.png)

- [ ] Boost page fractional (@wanderingstan working on fix now)
![image](https://user-images.githubusercontent.com/673455/55497729-383c2d00-55ff-11e9-87bd-513454d532b5.png)

- [ ] All dates, times, months, day names, etc in fractional/hourly fractional
![image](https://user-images.githubusercontent.com/673455/55497815-61f55400-55ff-11e9-85c3-acae711ce0b5.png)

**Comment by micahalcorn** [comment]
@wanderingstan should we just change this to use a loading indicator instead of text?

**Comment by wanderingstan** [comment]
@micahalcorn not sure I understand the question.... Oh, you mean the first one with "Loading..."!

I've talked with Aure and he likes the text but wants a spinner. Nick and I were talking about ideally having a screenful of gray placeholders, as Facebook and LinkedIn et al do.

**Comment by micahalcorn** [comment]
Yes, I like the placeholders. We used to do that on listing detail and cards back in the day, and I think Aure was the one that suggested it.

**Comment by micahalcorn** [comment]
And sorry @wanderingstan I somehow missed the second and third sub-issues here, so I've changed the title back.
